pkg/taskutil/taskutil.go

```diff
@@ -30,19 +30,38 @@ import (
 	"github.com/containerd/console"
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/cio"
+	"github.com/containerd/nerdctl/pkg/consoleutil"
 	"github.com/containerd/nerdctl/pkg/infoutil"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/term"
 )
 
 // NewTask is from https://github.com/containerd/containerd/blob/v1.4.3/cmd/ctr/commands/tasks/tasks_unix.go#L70-L108
 func NewTask(ctx context.Context, client *containerd.Client, container containerd.Container,
-	flagA, flagI, flagT, flagD bool, con console.Console, logURI string) (containerd.Task, error) {
+	flagA, flagI, flagT, flagD bool, con console.Console, logURI, detachKeys string) (containerd.Task, error) {
+	var t containerd.Task
+	closer := func() {
+		// t will be set by container.NewTask at the end of this function.
+		//
+		// We cannot use container.Task(ctx, cio.Load) to get the IO here
+		// because the `cancel` field of the returned `*cio` is nil. [1]
```

Comment:
it says that the io to be load for a task for `deletion`. So I think they assume the the task io is already canceled. But It is not intuitive at all we may load and then cancel  :/ [see](https://github.com/containerd/containerd/blob/8f756bc8c26465bd93e78d9cd42082b66f276e10/cio/io.go#L354-L355)

Reply:
That's true. Are you suggesting to open an issue at containerd side to make `cio.Load` also usable for cancel?

Reply:
Yes. I can take care of opening the issue if LGTY

Reply:
Sure, please go ahead, thanks! I can then link the issue as a code comment here.